schema/2.0.0/corpora_schema.md

```diff
@@ -0,0 +1,257 @@
+| organism | categorical with `str` categories | Data Portal |
+| sex | categorical with `str` categories | Data Portal |
+| tissue | categorical with `str` categories. This MUST be appended with `" (cell culture)"` or `" (organoid)"` if set in the matching identifier. | Data Portal |
+| | | |
+
+## `var` (Gene Metadata)
+
+`var` is a [`pandas.DataFrame`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html).
+
+`var.index` MUST contain unique ENSEMBL gene identifiers for features. 
+
+Curators MUST annotate the following columns in the `var` dataframe:
+
+| Key | Value | Annotator |
+:--|:--|:--
+| feature_biotype | This MUST be `"gene"`. | Data Portal |
+| feature_id (`var.index`) | `str`. This MUST be an ENSEMBL term from the gene annotation corresponding to the organism for the gene.  | Curator |
+||||
+
+When a dataset is uploaded, the cellxgene Data Portal MUST automatically add the matching human-readable name for the corresponding gene identifier to the `var` dataframe. Curators SHOULD NOT annotate the following column.
+
+| Key | Value | Annotator |
+:--|:--|:--
+| feature_name | `str`. This MUST be the ENSEMBL gene name corresponding to the `feature_id`. | Data Portal |
+||||
+
+## `obsm` (Embeddings)
+
+For each `str` key, `obsm` stores a `numpy.ndarray` of shape `(n_obs, m)`, where `n_obs` is the number of rows in `X` and `m >= 1`.
+
+To display a dataset in cellxgene Explorer, Curators MUST annotate **one or more** two-dimensional (`m >= 2`) embeddings (e.g. tSNE, UMAP, PCA, spatial coordinates) in `obsm`. The key for the embedding MUST be prefixed with `X_`. The text that follows this prefix is presented to users in the cellxgene Explorer *Embedding Choice* selector.
+
+To illustrate, the [Krasnow Lab Human Lung Cell Atlas, 10X dataset](https://cellxgene.cziscience.com/e/krasnow_lab_human_lung_cell_atlas_10x-1-remixed.cxg/) in the [A molecular cell atlas of the human lung from single cell RNA sequencing collection](https://cellxgene.cziscience.com/collections/5d445965-6f1a-4b68-ba3a-b8f765155d3a) defines two embeddings in `obsm`:
+
+* `'X_Compartment_tSNE'`
+* `'X_tSNE'`
+
+Users can then choose whether the 'Compartment_tSNE' or the 'tSNE' embedding  is visualized in cellxgene Explorer:
+
+![Embeddings Illustration](images/embeddings.png
+)
+
+See also `default_embedding` in `uns`.
+
+## `uns` (Dataset Metadata)
+
+`uns` is a ordered dictionary with a `str` key. Curators MUST annotate the following keys and values in `uns`:
+
+| Key | Value | Annotator |
+:--|:--|:--|
+| layer_descriptions | `dict` with `str` keys and values. Keys MUST be the layer names whose values are free text descriptions for how the layer was created (e.g. "counts per million"). One key MUST be `"X"` which describes the transformations (if any) performed to produce the `X` matrix that cellxgene Explorer displays. If a raw layer is present, the layer description value MUST be `"raw"`. | Curator |
```

Comment:
The assumption is correct, @jahilton would you recommend this to be clarified?

Reply:
I would say that there is an exception for ATAC/methylation seq. We don't require a raw layer, and thus the layer description should delineate how gene values were obtained with as much information as needed to the discretion of the curator.

Reply:
I think the existing wording leaves room for interpretation, but that's a good point about the different data types. The variety makes it difficult to be very specific. Perhaps some examples of what you expect to be included and at what level of detail would help.

Reply:
We've been having an internal discussion about the purpose of `layer_description`, what's it good for, and who consumes it, so Jason's comment validates that it's ambiguous to some degree - especially with the new requirements about the location of the raw matrix. And the use of "layer" does not match conceptually with the 'X' section which does not rely on layers.

@pablo-gar and @ambrosejcarr - can you offer a rewrite that clearly describes the rationale for this field and offers guidance for how curators should complete it?

Reply:
Rationale:
1. To identify which layer contains raw data (adata.X or adata.raw.X)
2. To increase re-usability of non-raw data when available, in this case the information of adata.uns["layer_descriptions"] can be use by users who download the data and wish to understand the representation of expression values. 

Rewriting: 
> `dict` with `str` keys and values. Keys MUST be the layer names whose values are free text descriptions for how the layer was created. One key MUST be `"X"`. If a layer has raw data its value must be "raw", otherwise the value SHOULD accurately summarize what the numbers in the layer represent (e.g.  "counts per million", "SCTransform normalization")

Reply:
With deprecation of `cellxgene-schema apply`, there are no longer any application dependencies on `layer_descriptions`. 

The location of `raw` is specified by the schema, so the only reason to include layer descriptions for raw would be if we felt files should be self-describing, and we could write `layer_descriptions` using the validator.

Except for X, it would be cleaner simply to say, in the matrix layers section, *additional layers SHOULD have names that accurately summarize what the numbers in the layer represent (e.g. "counts_per_million" or "SCTransform_normalized")*, rather than creating a dict to describe badly named layers. 

Given that, my recommendation is: 
* `layer_descriptions` is removed
* a RECOMMENDED dataset metadata `normalization` is added that describes the normalization approach for X. It could use a controlled vocabulary, if we have appetite for standardizing this right now. Imperfect text: *"`normalization` SHOULD describe the method used to normalize the data stored in X. If data in X are raw, this value SHOULD be `none`"*
* guidance is added to `X (matrix layers)`: *additional layers SHOULD have names that accurately summarize what the numbers in the layer represent (e.g. "counts_per_million", "SCTransform_normalized", or "RNA_velocity_unspliced")*

Reply:
Included in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/d473aa912d49748cd646c7312b7c19188a22a798